app/services/symphony_client.rb

```diff
@@ -34,10 +34,13 @@ def login_by_sunetid(sunetid)
 
   # get a session token by authenticating to symws
   def session_token
-    @session_token ||= begin
+    @session_token ||=
+      begin
```

Comment:
This indentation seems...odd.  Is rubocop complaining about something here?

Reply:
Yes, rubocop made me do it!

`rescue...is not aligned with begin`

Reply:
![yuck](https://media.giphy.com/media/xUA7bcUzBNSLC8Zy3C/giphy.gif)